open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Why title <code>Conditionals</code> is removed? If this is the case, you should make If-elif-else as a one level higher title. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p><code>If-elif-else</code> is more consistent with the naming convention for titles in <code>Python-programming-advanced.ipynb</code>.</p><p> </p><p> And it‘s already at a high level title <code>## If-elif-else</code></p><p>Should we add a sequence number before titles like <code>## 2.1 If-elif-else</code></p><p> </p>